services/brig/schema/src/V56.hs

```diff
@@ -0,0 +1,25 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes       #-}
+
+module V56 (migration) where
+
+import Imports
+import Cassandra.Schema
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 56 "Add table to exclude phone number prefixes" $ do
+    -- A table for manual excluding of phone number prefixes that abuse sms sending
+    -- and/or calling.
+    --
+    -- Operations we need to support:
+    --   * Add a new prefix of arbitrary length
+    --   * Remove a prefix
+    --   * Given a phone number, check whether it matches any existing prefix
+    void $ schema' [r|
```

Comment:
Sorry for only realizing it now, but I think it could be good to also add a "comment" field to the table (and even make it mandatory and non-empty). Otherwise in some time we might end up with a bunch of blacklisted prefixes in the database and no idea why or when they were blacklisted.

Reply:
https://github.com/wireapp/wire-server/pull/593/commits/6605fa765dc3dcf8f9cab922dca91b1fcea2c942